backend/agent/interactem/agent/agent.py

```diff
@@ -124,7 +125,18 @@ def __init__(self):
     async def _start_podman_service(self, create_process=False):
         self._podman_process = None
         if create_process:
-            args = ["podman", "system", "service", "--time=0", self._podman_service_uri]
+            if PodmanClient is PodmanHpcClient:
+                podman_cmd = "podman-hpc"
+            else:
+                podman_cmd = "podman"
```

Comment:
Isn't it always going to the `podman-hpc` unless you had some sort of conditional import?

Reply:
We have conditional import at line 74. We could also move this up to that, if that makes more sense

Reply:
But you are importing the hpc version at 21?

Reply:
that is a mistake

Reply:
fixed

Reply:
Oh that was not a mistake... that was needed to do the comparison. I will move this up to the conditional import